src/Table/Table.js

```diff
@@ -315,7 +315,7 @@ export default class Table extends React.Component {
     let buildRowOptions = this.props.buildRowOptions;
     let childToMeasure;
 
-    if (itemHeight === 0) {
+    if (itemHeight === 0 && data.length) {
```

Comment:
Before we made changes to this yesterday, the table component would render the regular table (as opposed to the `scrollTable`) if the length of the dataset were 0. We changed that because toggling between the `scrollTable` and the regular table prevented gemini from maintaining a reference to its container, which it needs to calculate its height. This worked, but `scrollTable` expects to measure the height of its first child (which is calculated by passing `data[0]` to the measuring function), and when the dataset is 0, there is no first item.